resilience4j-spring-boot2/src/test/java/io/github/resilience4j/circuitbreaker/monitoring/events/CircuitBreakerHystrixStreamEventsTest.java

```diff
@@ -0,0 +1,148 @@
+        }
+        Thread.sleep(1000); // to record the event
+        // The invocation is recorded by the CircuitBreaker as a success.
+        dummyService.doSomething(false);
+
+        CircuitBreakerEventsEndpointResponse circuitBreakerEventsAfter = circuitBreakerEvents(ACTUATOR_CIRCUITBREAKEREVENTS);
+        assertThat (circuitBreakerEventsBefore.getCircuitBreakerEvents().size() < circuitBreakerEventsAfter.getCircuitBreakerEvents().size());
+        Thread.sleep(1000); // for webClient to complete the subscribe operation
+        assertThat (events.size() == 2);
+    }
+
+    @Test
+    public void streamEventsbyName() throws IOException, InterruptedException {
+        List<ServerSentEvent<String>> events = getServerSentEvents(ACTUATOR_STREAM_CIRCUITBREAKER_EVENTS + "/backendA");
+        CircuitBreakerEventsEndpointResponse circuitBreakerEventsBefore = circuitBreakerEvents(ACTUATOR_CIRCUITBREAKEREVENTS + "/backendA");
+        // The invocation is recorded by the CircuitBreaker as a success.
+        dummyService.doSomething(false);
+        Thread.sleep(1000); // sleep is needed to record the event
+        try {
+            dummyService.doSomething(true);
+        } catch (IOException ex) {
+            // Do nothing. The IOException is recorded by the CircuitBreaker as part of the recordFailurePredicate as a failure.
+        }
+        CircuitBreakerEventsEndpointResponse circuitBreakerEventsAfter = circuitBreakerEvents(ACTUATOR_CIRCUITBREAKEREVENTS + "/backendA");
+        Thread.sleep(1000); //  webClient to complete the subscribe operation
+        assertThat (circuitBreakerEventsBefore.getCircuitBreakerEvents().size() < circuitBreakerEventsAfter.getCircuitBreakerEvents().size());
+        assertThat (events.size() == 2);
+    }
+
+    @Test
+    public void streamEventsbyNameAndType() throws IOException, InterruptedException {
+        List<ServerSentEvent<String>> events = getServerSentEvents(ACTUATOR_STREAM_CIRCUITBREAKER_EVENTS+ "/backendA/ERROR");
+        CircuitBreakerEventsEndpointResponse circuitBreakerEventsBefore = circuitBreakerEvents(ACTUATOR_CIRCUITBREAKEREVENTS + "/backendA");
+        // The invocation is recorded by the CircuitBreaker as a success.
+        dummyService.doSomething(false);
+        Thread.sleep(1000); // to record the event
+        try {
+            dummyService.doSomething(true);
+        } catch (IOException ex) {
+            // Do nothing. The IOException is recorded by the CircuitBreaker as part of the recordFailurePredicate as a failure.
+        }
+        CircuitBreakerEventsEndpointResponse circuitBreakerEventsAfter = circuitBreakerEvents(ACTUATOR_CIRCUITBREAKEREVENTS + "/backendA");
+        Thread.sleep(1000); // for webClient to complete the subscribe operation
+        assertThat (circuitBreakerEventsBefore.getCircuitBreakerEvents().size() < circuitBreakerEventsAfter.getCircuitBreakerEvents().size());
+        assertThat (events.size() == 1);
```

Comment:
Assertj has a fluent API. You have to do
`assertThat(events.size()).isEqulTo(1);`

Reply:
Oh.. yeah. I see.. I updated it and pushed it

Reply:
@RobWin